pages/api/blog.js

```diff
@@ -0,0 +1,19 @@
+import fetch from 'isomorphic-fetch'
+
+export default async (_, res) => {
+  try {
+    const response = await fetch(`https://blog.dvc.org/api/posts.json`)
+
+    if (response.status !== 200) {
+      res.status(502).json({ error: 'Unexpected response from Blog' })
+
+      return
+    }
+
+    const data = await response.text()
+
+    res.status(200).json(data)
+  } catch {
+    res.status(404)
```

Comment:
Is this the appropriate status code? Maybe 503/502 is better since it's an upstream error.

Reply:
You are right, will change.

Reply:
You are right, will change it.